platform/cri-o/crio.conf.upstream

```diff
@@ -152,6 +153,7 @@ additional_devices = [
 
 # Path to OCI hooks directories for automatically executed hooks.
 hooks_dir = [
+	"/usr/share/containers/oci/hooks.d", 
```

Comment:
What do we put in this hooks directory?

Reply:
That's the upstream file (to detect upstream config file generation changes). The array is empty in `crio.conf.in`.

Reply:
Oh, yeah, misread that. :heavy_check_mark: